FRENCH/src/ch20-01-single-threaded.md

```diff
@@ -398,12 +398,12 @@ characters for characters in the buffer that aren’t filled by request data.
 
 Ensuite, nous convertissons les octets présents dans le tampon en chaînes de
 caractères et nous affichons cette chaîne de caractères. La fonction
-`String::from_utf8_lossy` prend en paramètre un `&[u8]` et le transforme en une
+`String::from_utf8_lossy` prends en paramètres un `&[u8]` le transforme en une
```

Comment:
```suggestion
`String::from_utf8_lossy` prends en paramètre un `&[u8]` et le
```

I don't understand what's wrong with this ?

Reply:
grammatical error 'prendS', a single parameter so adding a s is useless but not wrong and a 'et' is missing for the last action performed ...

Reply:
Yeah I agree with you, so your suggestion to add an `s` to `paramètre` and also removing the `et` word is not what you want to do, right ?

Reply:
yes you're right. Have I made a mistake again :-( ?

Reply:
Don't worry, *shit happens* ! 😄

I let you commit my suggestion, if everything is now OK for you ? 😉

Reply:
you have to also remove the s after "prend": I don't know how to do that here ...

Reply:
oh, OK ! 😅
There :

```suggestion
`String::from_utf8_lossy` prend en paramètre un `&[u8]` et le transforme en une
```

Reply:
Here we are ! Thanks :-)

Reply:
I think this is one of the conflicts with #359 I tell just above 😉